pytest.ini.ign

```diff
@@ -0,0 +1,8 @@
+[pytest]
```

Comment:
unrelated. please remove. Consider to put it into a separate PR.

Reply:
I already did. they belong to separate commit or PR. submitted by accident btw